guides/release/upgrading/editions.md

```diff
@@ -0,0 +1,1078 @@
+Editions are a new concept that were recently introduced to Ember. An edition
```

Comment:
A quick observation – it's probably useful to split this introduction to Editions apart from the **Features** section. They have different potential audiences, especially for people who may come in via media pushes.